src/main/java/com/ldbc/driver/workloads/ldbc/snb/bi/LdbcSnbBiQuery10TagPerson.java

```diff
@@ -29,6 +31,14 @@ public int limit()
         return limit;
     }
 
+    @Override
+    public Map<String, Object> parameterMap() {
```

Comment:
how is a user supposed to know what these keys are?

same question for all other Operations